cl-data-structures.asd

```diff
@@ -0,0 +1,32 @@
+(in-package #:cl-user)
+
+(asdf:defsystem cl-data-structures
+  :name "cl-data-structures"
+  :version "0.0.0"
+  :license "MIT"
+  :author "Lisp Mechanics"
+  :maintainer "Lisp Mechanics"
+  :depends-on (:iterate :alexandria :serapeum)
+  :serial T
+  :components ((:file "package")
```

Comment:
src should be added as system's pathname. Module name should be a string; it serves then as a pathname to the subsystem(so, cl-data-structures system's pathname "src" and module name "api" would imply that the pathname of its components is "src/api").